ts/components/ui/ListItemTransaction.tsx

```diff
@@ -0,0 +1,210 @@
+import { WithTestID } from "../../types/WithTestID";
+import { IOBadge } from "../core/IOBadge";
+import { Icon } from "../core/icons";
+import { IOLogoPaymentType, LogoPayment } from "../core/logos";
+import { VSpacer } from "../core/spacer/Spacer";
+import { LabelSmall } from "../core/typography/LabelSmall";
+import { NewH6 } from "../core/typography/NewH6";
+import { useIOTheme } from "../core/variables/IOColors";
+import {
+  IOListItemStyles,
+  IOListItemVisualParams,
+  IOStyles,
+  IOVisualCostants
+} from "../core/variables/IOStyles";
+import Avatar from "./Avatar";
+import {
+  PressableBaseProps,
+  PressableListItemBase
+} from "./utils/baseListItems";
+
+type LogoNameOrUri = IOLogoPaymentType | ImageURISource;
+export type ListItemTransaction = WithTestID<
+  PressableBaseProps & {
+    hasChevronRight?: boolean;
+    isLoading?: boolean;
+    paymentLogoOrUrl?: LogoNameOrUri;
+    subtitle: string;
+    title: string;
+  } & (
+      | {
+          transactionStatus: "success";
+          transactionAmount: string;
+        }
+      | {
+          transactionStatus: "failure" | "pending";
+          transactionAmount?: string;
+        }
+    )
+>;
+
+type LeftComponentProps = {
+  logoNameOrUrl: LogoNameOrUri;
+};
+
+const isImageUrI = (
+  value: IOLogoPaymentType | ImageURISource
+): value is ImageURISource =>
+  typeof value === "object" && value.uri !== undefined;
+
+const LeftComponent = ({ logoNameOrUrl }: LeftComponentProps) => {
+  if (isImageUrI(logoNameOrUrl)) {
+    return <Avatar shape="circle" size="small" logoUri={[logoNameOrUrl]} />;
+  } else {
+    return (
+      <View
+        style={{
+          width: IOVisualCostants.avatarSizeSmall,
+          height: IOVisualCostants.avatarSizeSmall,
+          alignItems: "center",
+          justifyContent: "center"
+        }}
+      >
+        <LogoPayment name={logoNameOrUrl} />
+      </View>
+    );
+  }
+};
+
+export const ListItemTransaction = ({
+  accessibilityLabel,
+  hasChevronRight = false,
+  isLoading = false,
+  paymentLogoOrUrl,
+  onPress,
+  subtitle,
+  testID,
+  title,
+  transactionAmount,
+  transactionStatus = "success"
+}: ListItemTransaction) => {
+  const theme = useIOTheme();
+
+  const isDSEnabled = useIOSelector(isDesignSystemEnabledSelector);
+  if (isLoading) {
+    return <SkeletonComponent />;
+  }
+
+  const designSystemBlue = isDSEnabled ? "blue" : "blueIO-500";
+  const ListItemTransactionContent = () => {
+    const TransactionAmountOrBadgeComponent = () => {
+      switch (transactionStatus) {
+        case "success":
+          return (
+            <NewH6 color={hasChevronRight ? designSystemBlue : "black"}>
+              {transactionAmount || "-"}
+            </NewH6>
+          );
+
+        case "failure":
+          return (
+            <IOBadge
+              color="red"
+              variant="solid"
+              labelColor="bluegreyDark"
+              text={I18n.t("global.badges.failed_f").toUpperCase()}
```

Comment:
```suggestion
              text={I18n.t("global.badges.failed").toUpperCase()}
```

Reply:
Btw, the `toUpperCase()` should be removed because the visual appearance is managed through the component style. [See the reference](https://github.com/pagopa/io-app/blob/cd933b94598ae9f7b24050d5f61c599d114c6465/ts/components/core/Tag.tsx#L80) mentioned above.

Reply:
I didn't notice the `toUpperCase`. You are right. I updated the suggestion!

Reply:
adressed in [c6627d7](https://github.com/pagopa/io-app/pull/4710/commits/c6627d7de10649bfc65ca3523cf44e9ff6be1df3)